packages/cardano-services/package.json

```diff
@@ -57,7 +61,7 @@
     "test:debug": "DEBUG=true yarn test",
     "test:e2e": "shx echo 'test:e2e' command not implemented yet",
     "test:load": "jest -c ./load.jest.config.js --runInBand",
-    "testnet:up": "DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 OGMIOS_PORT=1338 NETWORK=testnet docker-compose -p cardano-services-testnet up",
+    "testnet:up": "DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 OGMIOS_PORT=1338 NETWORK=testnet CARDANO_NODE_VERSION=1.35.2 docker-compose -p cardano-services-testnet up",
```

Comment:
Why not `1.35.3`?

Reply:
That's the last compatible version with _testnet_

Reply:
I missed a required change to make this ENV work: https://github.com/input-output-hk/cardano-js-sdk/pull/405/commits/0387afaf71a9a0863fe91a07b9efb370796cb1e5